packages/contracts-bedrock/contracts/deployment/MigrationSystemDictator.sol

```diff
@@ -273,6 +279,11 @@ contract MigrationSystemDictator is BaseSystemDictator {
             L1ChugSplashProxy(payable(config.proxyAddressConfig.l1StandardBridgeProxy)).setOwner(
                 address(config.globalConfig.finalOwner)
             );
+
+            // Transfer ownership of the L1ERC721Bridge to the final owner.
+            Proxy(payable(config.proxyAddressConfig.l1ERC721BridgeProxy)).changeAdmin(
+                address(config.globalConfig.finalOwner)
```

Comment:
SHouldn't the final owner be the proxy admin, which is owned by the final owner?

Reply:
This is an exit codepath that is meant to reset the system back to its original state. At this point the proxies have not yet been transferred to the proxy admin, so I figured it was best to return back to the original admin.